jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -305,6 +309,29 @@ public JibContainerBuilder addLabel(String key, String value) {
     return this;
   }
 
+  /**
+   * Sets the format to build the container image as. Use {@link ImageFormat#Docker} for Docker V2.2
```

Comment:
The `as` is dangling.  `Sets the build container image format.`

Reply:
The `as` here is referring to the `JibContainerBuilder builds the container image as the format specified` - perhaps `Sets the container image format to build as`?

Reply:
I agree ending the sentence with "as" is a bit awkward.